website1/src/scss/_components.scss

```diff
@@ -91,14 +91,12 @@
 }
 
 .title {
-  width: 50%;
-  margin-left: 0;
-  padding-top: 75px;
   color: $font-color-grey;
 
   &__header {
     text-transform: uppercase;
     font-size: 36px;
+    padding-top: 100px;
```

Comment:
Are you confident that this doesn't break the design?

Reply:
Yes, of course.